test/test_jit.py

```diff
@@ -16414,6 +16414,12 @@ def keys(x):
 
         self.assertEqual(set(keys(self.dict())), set(self.dict().keys()))
 
+        @torch.jit.script
+        def specialized_list():
+            return {1: 1, 2: 2}.keys()
+
+        self.assertTrue(set(specialized_list()) == set([1, 2]))
```

Comment:
This would have passed with `deprecatedUntypedList` before, how does it test the change?

Reply:
now it is testing an op with a specialized list. i don't know what `deprecatedUntypedList` is

Reply:
okay added op that uses specialized list